system-addon/lib/ActivityStream.jsm

```diff
@@ -133,6 +134,10 @@ const PREFS_CONFIG = new Map([
   ["section.topstories.showDisclaimer", {
     title: "Boolean flag that decides whether or not to show the topstories disclaimer.",
     value: true
+  }],
+  ["tippyTop.service.endpoint", {
+    title: "Tippy Top service manifest url",
+    value: "https://s3.amazonaws.com/activitystream-dev-default-resources-s3bucket-1qw8m6s29v3dq/tippytop/icons.json"
```

Comment:
@Mardak How is this pref managed later on once it lands? We might want separate values for nightly/beta/release. We might want to run experiments with different URLs. etc. Do I need to do anything special for now?

Reply:
If it's via a shield study, the prefs will be replaced with some other value. Also, we probably need to fix up mozilla-central test prefs to disable this. And what's the plan for production endpoint?